File: Modul 5/Soal 4/PRAK504-NIM-Nama.py
def reverse(a):
    b = 0
    
    while a != 0:
        last = a % 10
        b = b * 10 + last
        a = a//10
    return b

A, B = map(int, input().split())

A = reverse(A)
B = reverse(B)

C = A + B

print(f"{reverse(C)}")

File: Modul 5/Soal 4/PRAK504-NIM-Nama.c
#include <stdio.h>

int reverse(int a){
    int b = 0;

    while(a != 0){
        int last = a % 10;
        b = b * 10 + last;
        a = a/10;
    }
    return b;
}

int main(){
    int A, B;
    
    scanf("%d %d", &A, &B);

    A = reverse(A);
    B = reverse(B);

    int C = A + B;

    printf("%d", reverse(C));
}